src/Facebook/GraphNodes/GraphAlbum.php

```diff
@@ -28,19 +28,19 @@
  *
  * @package Facebook
  */
-
 class GraphAlbum extends GraphNode
 {
     /**
-     * @var array Maps object key names to Graph object types.
+     * @var array Maps object key names to GraphNode types.
      */
     protected static $graphObjectMap = [
-        'from' => '\Facebook\GraphNodes\GraphUser',
```

Comment:
This is technically a BC. Probably not likely someone is really relying on getting a `GraphUser` in the `from` field though. And it's certainly being cast as the wrong subtype as a `GraphUser` so it should be changed. @yguedidi Thoughts?